sdk/communication/azure-communication-identity/azure/communication/identity/_shared/models.py

```diff
@@ -12,7 +12,9 @@ class DeprecatedEnumMeta(CaseInsensitiveEnumMeta):
 
     def __getattribute__(cls, item):
         if item == "MICROSOFT_BOT":
-            warnings.warn("MICROSOFT_BOT is deprecated and should not be used.", DeprecationWarning)
+            warnings.warn("MICROSOFT_BOT is deprecated and has been replaced by \
+                          MICROSOFT_TEAMS_APP identifier.", DeprecationWarning)
+            return CaseInsensitiveEnumMeta.__getattribute__(cls, "MICROSOFT_TEAMS_APP")
```

Comment:
Can we make sure we have a test for the backwards compatibility of the enum?

Reply:
I have reverted the change to return the old value of enum.